Makefile

```diff
@@ -23,18 +35,23 @@ export DOCKER_BUILDKIT:=1
 help:
 	@echo "jupyter/docker-stacks"
 	@echo "====================="
-	@echo "Replace % with a stack directory name (e.g., make build/minimal-notebook)"
+	@echo "Replace % with a stack directory name (e.g., make build-cross/minimal-notebook)"
 	@echo
 	@grep -E '^[a-zA-Z0-9_%/-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-build/%: DARGS?=
-build/%: ## build the latest image for a stack
-	docker build $(DARGS) --rm --force-rm -t $(OWNER)/$(notdir $@):latest --build-arg OWNER=$(OWNER) ./$(notdir $@)
+build-x86/%: DARGS?=
+build-x86/%: ## build the latest image for a stack on x86 only
+	docker buildx build $(DARGS) --rm --force-rm -t $(OWNER)/$(notdir $@):latest ./$(notdir $@) --build-arg OWNER=$(OWNER) --platform "linux/amd64" --push
```

Comment:
I think you need to include the `$REGISTRY` environment variable as part of the tag argument so that it gets pushed to `localhost:5000`